src/js/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { getDateFormatted, getPercentFormatted } from '../utils/helpers';
+
+class UserInfoSection extends React.Component {
+  render() {
+    const { userData } = this.props;
+
+    // Used to get today's date to show information current of
+    const today = new Date();
+    const todayFormatted = getDateFormatted(today);
+
+    // Check if percent is a number
+    let percentageBenefit = 'unavailable';
+    if (getPercentFormatted(userData.percentageBenefit)) {
+      percentageBenefit = getPercentFormatted(userData.percentageBenefit);
+    }
+
+    return (
+      <div className="gibstatus">
+        <h3 className="section-header">{userData.firstName} {userData.lastName}</h3>
+        <div className="usa-alert usa-alert-info">
+          <div className="usa-alert-body">
+            <h4 className="usa-alert-heading">This information is current as of {todayFormatted}</h4>
```

Comment:
I had a question on the mocks for this: this notice isn't shown on the view where there is actual enrollment history data available (only when either the user is not eligible, or there is no data available). @phiden is that by design or should we always show this notice?